travis/5.6.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+yes | pecl install -f amqp-1.6.1
```

Comment:
Same here, php-amqp support PHP >= 5.3 (before 1.7.0 even 5.2 was known to be compatible, though we removed it from build matrix as too outdated).